test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -537,7 +537,7 @@ def test_pip_check(image):
     # TF2.9 sagemaker containers introduce tf-models-official which has a known bug where in it does not respect the
     # existing TF installation. https://github.com/tensorflow/models/issues/9267. This package in turn brings in
     # tensorflow-text. Skip checking these two packages as this is an upstream issue.
-    if framework == "tensorflow" and Version(framework_version) in SpecifierSet(">=2.9.1"):
+    if framework in ["tensorflow", "huggingface_tensorflow"]  and Version(framework_version) in SpecifierSet(">=2.9.1"):
```

Comment:
nit: this condition looks like it is need of some refactoring - will this also apply to inference images?

Reply:
I added huggigface tensorflow to the same condition because the issue pertains to both training and inference for tensorflow